ecl/key/ClassificationTestModified.xml

```diff
@@ -0,0 +1,5 @@
+<Result>
```

Comment:
Please remove all '<Result>\n' and </Result>\n' lines from both key file as well.